data/human/human missions.txt

```diff
@@ -3601,3 +3601,58 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+	
+mission "Thrall Emigration"
+	name "Thrall Emigration"
+	description "Transport a family from Thrall to <destination>, where they hope to find better employment. They cannot pay you."
+	minor
+	source
+		planet "Thrall"
+	destination "Delve"
+	to offer
+		random < 50
+	passengers 2
+	on offer
+		conversation
+			`By the edge of the spaceport at night, you see a thin sweaty man turn away from a pilot, disheartened. You make eye contact, and he begins hurrying over to you.`
+			choice
+				`	(Wait for him.)`
+				`	(Walk away quickly.)`
+					decline
+			`	"Excuse me! Are you a pilot?" Without waiting for a response, he continues "I need to go to the Syndicate. I work all day picking cotton with these hands, but my family's still starving. We can't afford transport..."`
+			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes are human.`
+			choice
+				`	"I might be able to help you. What planet do you need to reach?"`
+				`	"I'm sorry, I only transport for pay."`
+					goto disappointment
+			`	His eyes brighten with hope. "Oh! I, I heard some people going to Delve to be miners. They hire folk like us. I, if you could get me, and my wife and our two sons there, I'd praise you to the high stars. You'd save our lives."`
+			choice
+				`	"I can take you, but no more."`
+					goto separation
+				`	"I'd be happy to offer you four transport."`
+					goto offer
+				`	"Sorry, I think this won't work out."`
+					goto disappointment
+			label separation
+			`	"The two little ones are really small," he persists. "We can cramp together. You'll just notice two of us. I mean, some go themselves and send money back, but..."`
+			`	He shakes his head. "My wife's already told me we need to stay together, and I agree. I can't leave. Please, nobody else has said yes."`
+			choice
+				`	"I guess I can find room."`
+					goto offer
+				`	"Sorry, but that's still too many. Have a nice day."`
+					goto disappointment
+			label offer
+			`	"Oh, thank you, thank you!" he says, vigorously shaking your hand. "I'll get them fast, we can leave tonight! I'm Gaint, by the way" He quickly disappears, turning back at least three times to tip his hat to you and make sure you haven't left.`
+			`	When Gaint's family arrives, your heart warms. His wife, Mell, is carrying their infant child, and a toddler clutches on her leg. His fingers look like a skeleton, and his eyes look like his dad's. "Thank you so much," Mell says as you load them onto the ship.`
```

Comment:
The phrase 'your heart warms' is kind of corny, and moreover dialogs aren't supposed to tell the player how they feel.

Reply:
I like it when it suggest emotions but maybe it should not be so affirmative of what you feel

Reply:
```suggestion
			`	Gaint's family arrives quickly. His wife, Mell, is carrying their infant child, and a toddler clutches her leg. His fingers look like a skeleton, and his eyes look like his dad's. "Thank you so much," Mell says as they enter your ship.`
```
Agreed, thanks for pointing that out. Changed a few other words as well. How does this version read?

Reply:
That's a lot better, IMO.